markdowns/19_Text_to_Image_search_pipeline_with_MultiModal_Retriever.md

```diff
@@ -0,0 +1,190 @@
+---
+layout: tutorial
+colab: https://colab.research.google.com/github/deepset-ai/haystack-tutorials/blob/main/tutorials/19_Text_to_Image_search_pipeline_with_MultiModal_Retriever.ipynb
+toc: True
+title: "Text-to-Image search pipeline with MultiModal Retriever"
+last_updated: 2022-11-03
+level: "intermediate"
+weight: 95
+description: Use a MultiModalRetriever to build a cross modal search pipeline.
+category: "QA"
+aliases: ['/tutorials/multimodal']
+---
+    
+
+**Level**: Intermediate
+
+**Time to complete**: 20 minutes
+
+**Prerequisites**: Prepare the Colab environment (see links below).
+
+**Nodes Used**: InMemoryDocumentStore, MultiModalRetriever
+
+**Goal**: After completing this tutorial, you will have learned about the MultiModalRetriever, and built a simple retrieval pipeline that searches for relevant images given a text query.
+
+To make this pipeline work, we first need to embed the images from the dataset using a transformer model. These image embeddings are representative of the respective image's content. These embeddings lie in a high-dimensional vector space. Then we embed the text query using the same transformer model (i.e. OpenAI CLIP). Because we use the same transformer model, these text embeddings also exist in the same space as the image embeddings. Finally, we perform a nearest neighbor search to retrieve relevant images for our text query.
```

Comment:
To me, this belongs to the tutorial description. It's not really a goal. 

If we move this to the description, this needs a bit of rephrasing (and I'd simplify it a bit too):
In this tutorial, you'll download a set of images that you'll then turn into embeddings using a transformers model, OpenAI CLIP. You'll then use the same model to embed the text query. Finally, you'll perform a nearest neighbor search to retrieve the images relevant to the text query.

I woulnd't mention the high-dimentional space, etc. as I don't think it's crucial to understand/complete the tutorial and it may be confusing at this point.

Reply:
btw. I added the same comments to Branden's tutorials :) I think we should align all of them to give the user a consistent experience so that they know what to expect.

Reply:
Changed :)